cqf-fhir-cr/src/main/java/org/opencds/cqf/fhir/cr/measure/r4/R4CareGapStatusEvaluator.java

```diff
@@ -52,12 +68,15 @@ public CareGapsStatusCode getGapStatus(Measure measure, MeasureReport measureRep
                     && population.getCount() == 1) {
                 inDenominator.setValue(true);
             }
-        }));
+        });
 
         // default improvementNotation
         boolean isPositive = true;
 
-        // if value is present, set value from measure if populated
+        // TODO: look for group specified 'improvement notation', if missing, then look on measure
```

Comment:
Do you plan to address this TODO?

Reply:
I am addressing in a separate merge, it was believed this was already integrated.